themes/edx.org/lms/templates/header.html

```diff
@@ -69,10 +76,9 @@ <h2 class="course-header">
 
     % if user.is_authenticated():
       % if not course or disable_courseware_header:
-        % if not nav_hidden or show_program_listing:
           <nav aria-label="Main" class="nav-main">
             <ul class="left list-inline nav-global authenticated">
-              % if not nav_hidden:
+              % if not nav_hidden and not show_program_listing:
                 <%block name="navigation_global_links_authenticated">
```

Comment:
@schenedx I have updated this condition later because `nav_hidden` is `True` on dashboard so both the navigation header and tabs (programs and courses) were also not shown on leaner's dashboard. Please verify this on [sandbox](https://tasawernawaz.sandbox.edx.org/) and let me know if you have any concerns.

FYI: @zubair-arbi